RCNN_tf.py

```diff
@@ -104,97 +81,80 @@ def rcl(input_data, num_input_chans, num_filter, filter_shape, num_of_data,
         padding='SAME'
     )
 
-    if "CPU" in processing_unit.upper():
```

Comment:
Nice catch that we do not have to specify anymore whether we are using a GPU or a CPU!